src/index.js

```diff
@@ -24,6 +24,18 @@ const sugar = {
   }
 }
 
+const _60fps = 1000/60
+
+export const rAF = typeof requestAnimationFrame !== 'undefined' ?
+  requestAnimationFrame :
+  (async (callback) => new Promise((resolve) => setTimeout(() => { resolve(); callback() }, _60fps)))
+if (typeof requestAnimationFrame === 'undefined') console.error('requestAnimationFrame not found - falling back to 60 fps')
```

Comment:
took some inspiration from css-tricks: https://css-tricks.com/using-requestanimationframe

Reply:
let's keep it simple and roll with just

```js
export const rAF = typeof requestAnimationFrame !== 'undefined' && requestAnimationFrame
export const rIC = typeof requestIdleCallback !== 'undefined' && requestIdleCallback
```

just a small shorthand proxy

Reply:
hmmm the tests will need to update too - lemme know if you're fine with the updated tests (optional chaining requires a plugin)